plugins/services/src/js/containers/service-detail/ServiceDetail.js

```diff
@@ -185,7 +187,10 @@ class ServiceDetail extends mixin(TabsMixin) {
     if (instanceCount > 0) {
```

Comment:
I think instead of hiding with a classname, we should not push the action if SDK. We can add that check to this `if` statement. We can then get rid of the new `prop` required on `PageHeaderActionsMenu`.